### PR TITLE
Dropping a separate NPDTstruct12 object and instead use NPDTstruct52

### PR DESCRIPTION
**Pull request by AHSauge**
NPDTstruct12 is now only used when loading and saving. Turning auto calc
on and off now no longer switches between to different set of values. This should fix bug #3278

### CODE REVIEW

File: components/esm/loadnpc.cpp [hunk 1]
```diff
@@ -62,12 +62,23 @@ namespace ESM
                     if (esm.getSubSize() == 52)
                     {
                         mNpdtType = NPC_DEFAULT;
-                        esm.getExact(&mNpdt52, 52);
+                        esm.getExact(&mNpdt, 52);
                     }
                     else if (esm.getSubSize() == 12)
                     {
+                    	//Reading into temporary NPDTstruct12 object
```
Comment (akortunov) [inline components/esm/loadnpc.cpp]: Please, use space characters onstread of tabs (4 space = 1 tab).

File: components/esm/loadnpc.cpp [hunk 2]
```diff
@@ -135,9 +146,19 @@ namespace ESM
...
 
         if (mNpdtType == NPC_DEFAULT)
-            esm.writeHNT("NPDT", mNpdt52, 52);
+        {
+        	esm.writeHNT("NPDT", mNpdt, 52);
+        }
         else if (mNpdtType == NPC_WITH_AUTOCALCULATED_STATS)
-            esm.writeHNT("NPDT", mNpdt12, 12);
+        {
+        	NPDTstruct12 npdt12;
```
Comment (akortunov) [inline components/esm/loadnpc.cpp]: The same thing is here.

**Comment by akortunov** [comment]
The full list of tabs:

Error: Tab characters found!
components/esm/loadnpc.cpp:69: //Reading into temporary NPDTstruct12 object components/esm/loadnpc.cpp:70: NPDTstruct12 npdt12;
components/esm/loadnpc.cpp:150: esm.writeHNT("NPDT", mNpdt, 52);
components/esm/loadnpc.cpp:154: NPDTstruct12 npdt12;
components/esm/loadnpc.cpp:155: npdt12.mLevel = mNpdt.mLevel;
components/esm/loadnpc.cpp:156: npdt12.mDisposition = mNpdt.mDisposition; components/esm/loadnpc.cpp:157: npdt12.mReputation = mNpdt.mReputation; 
components/esm/loadnpc.cpp:158: npdt12.mRank = mNpdt.mRank;
components/esm/loadnpc.cpp:160: esm.writeHNT("NPDT", npdt12, 12);

**Comment by Capostrophic** [comment]
You'll know you've dealt with tabs once Travis build stops failing - there are some tests to prevent tabs from getting into the code base even if everything's built fine.

**Comment by AHSauge** [comment]
Sorry, I seem to have forgotten to switch Eclipse away from using tab in that workspace. I believe my latest commit should solve that.

**Pull request merged**